packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenuItemWrapper/ContextualMenuSplitButton.tsx

```diff
@@ -151,20 +151,20 @@ export class ContextualMenuSplitButton extends ContextualMenuItemWrapper {
       dismissMenu
     } = this.props;
 
-    const itemProps = {
+    const itemProps: IContextualMenuItem = {
       onClick: this._onIconItemClick,
       disabled: isItemDisabled(item),
       className: classNames.splitMenu,
       subMenuProps: item.subMenuProps,
       submenuIconProps: item.submenuIconProps,
       split: true,
       key: item.key
-    } as IContextualMenuItem;
+    };
 
-    const buttonProps = assign({}, getNativeProps(itemProps, buttonProperties), {
+    const buttonProps = assign({}, getNativeProps<React.ButtonHTMLAttributes<HTMLButtonElement>>(itemProps, buttonProperties), {
```

Comment:
Another minor improvement: just use the spread operator rather than `assign`

Reply:
Sure..

Reply:
Done in https://github.com/OfficeDev/office-ui-fabric-react/pull/8462/commits/b1036d3cc2ffc08873af2ea0a2d13e9cd8b0e417.

Reply:
Even saved some bytes 👍. Great suggestion!